Queries and Scheduled Queries UI: Performance impact tooltips

RachelElysia opened the pull request:
Cerra #2750

Note: #344 in this sprint will be refactoring these tooltips to fresher code and UI

Tooltips added: 
1. Host details page Performance impact tooltip and Undetermined tooltip (custom tooltip wording compared to the other 3 pages)
2. Queries page Performance impact tooltip and Undetermined tooltip
3. Schedule page Performance impact tooltip and Undetermined tooltip
4. Packs page Performance impact tooltip and Undetermined tooltip

Respective order:
<img width="1290" alt="Screen Shot 2022-02-14 at 2 44 42 PM" src="https://user-images.githubusercontent.com/71795832/153956088-bb5e3606-d6fe-4da0-a90a-4d8c2185cbd1.png">
<img width="1292" alt="Screen Shot 2022-02-14 at 2 45 07 PM" src="https://user-images.githubusercontent.com/71795832/153956086-bbd821f9-4bf9-4bea-83aa-e885eaf5be8d.png">
<img width="1291" alt="Screen Shot 2022-02-14 at 2 44 24 PM" src="https://user-images.githubusercontent.com/71795832/153956090-87650435-c8a9-4c32-9f7b-c88216c25a09.png">
<img width="1291" alt="Screen Shot 2022-02-14 at 2 44 16 PM" src="https://user-images.githubusercontent.com/71795832/153956092-d7ab47ca-b158-4d41-85c1-e5e8ecc49ff0.png">
<img width="1289" alt="Screen Shot 2022-02-14 at 2 54 37 PM" src="https://user-images.githubusercontent.com/71795832/153956080-2e6aa33e-f56f-4891-9531-86897c2a50af.png">
<img width="1289" alt="Screen Shot 2022-02-14 at 2 54 46 PM" src="https://user-images.githubusercontent.com/71795832/153956075-a17b3c76-6d4a-4aa5-ba5f-114c5a4446a6.png">
<img width="1292" alt="Screen Shot 2022-02-14 at 2 53 41 PM" src="https://user-images.githubusercontent.com/71795832/153956084-818ac556-665f-4fe8-bb21-bee87048147f.png">
<img width="1284" alt="Screen Shot 2022-02-14 at 2 53 50 PM" src="https://user-images.githubusercontent.com/71795832/153956082-6c24178b-4af3-489d-9926-c8fc81d03266.png">



# Checklist for submitter

If some of the following don't apply, delete the relevant line.

- [x] Changes file added (for user-visible changes)
- [x] Manual QA for all new/changed functionality

lukeheath reviewed (approved):
Looks good! 👍  Thanks for catching the additional tables.

lukeheath reviewed (changes requested):
Just noticed an alignment issue with the "Actions" button in the schedules table: 

![image](https://user-images.githubusercontent.com/2495927/153966089-7ae5e54a-07e1-486c-bffe-7fcb66b2a731.png)

I checked `main` and it only appears to be happening on this branch.

RachelElysia commented:
> Just noticed an alignment issue with the "Actions" button in the schedules table:
> 
> ![image](https://user-images.githubusercontent.com/2495927/153966089-7ae5e54a-07e1-486c-bffe-7fcb66b2a731.png)
> 
> I checked `main` and it only appears to be happening on this branch.

On it!

RachelElysia commented:
Fixed, changed span to be targetted using classname `.queries-table__performance-impact-header` so it won't mess with the dropdown which also uses span.

lukeheath reviewed (approved):
Thanks!
